eval_type_backport/eval_type_backport.py

```diff
@@ -179,15 +179,31 @@ def _evaluate(
         self,
         globalns: dict[str, Any] | None,
         localns: Mapping[str, Any] | None,
+        type_params: (
+            tuple[typing.TypeVar | typing.ParamSpec | typing.TypeVarTuple, ...]
+            | frozenset[str]
+            | None
+        ) = None,
         recursive_guard: frozenset[str] | None = None,
```

Comment:
```suggestion
        *args: Any, **kwargs: Any
```

and then pass those through, much simpler and safer. I don't think this version of `ForwardRef` needs to be particularly user-friendly or well-hinted, and accurate hints depend on the Python version anyway. @bswck wdyt?

Reply:
@PerchunPak if you're happy with this suggestion then you can go ahead with it.

Reply:
> and then pass those through, much simpler and safer. I don't think this version of `ForwardRef` needs to be particularly user-friendly or well-hinted, and accurate hints depend on the Python version anyway. @bswck wdyt?

Agree

Reply:
We can `@functools.wraps(ForwardRef._evaluate)` in case we want to be pedantic

Reply:
Great idea, thanks!